docs/network/architecture.md

```diff
@@ -51,12 +66,16 @@ For more information about Tezos layer 1 and its nodes, see [Architecture](https
 
 ## Transaction lifecycle
 
-The overall lifecycle of a typical operation is as follows:
+Etherlink has a standard method of processing transactions and a backup method that protects it from censorship and
```

Comment:
missing end of sentence

Reply:
Completed it as "protects it from censorship and network problems."